src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParametersTabs.js

```diff
@@ -20,19 +21,30 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function _buildScenarioTabList(tabs, userRoles, t) {
+function _buildScenarioTabList(tabs, userRoles, t, errors) {
   const tabListComponent = [];
+  const errorsByTab = Object.assign(
+    {},
+    ...tabs.map((tab) => ({
+      [tab.id]: tab.parameters.map((param) => param.id).filter((parameter) => Object.keys(errors).includes(parameter)),
+    }))
+  );
```

Comment:
Why don't directly do:
`const errorsByTab = {
    ...tabs.map((tab) => ({
      [tab.id]: tab.parameters.map((param) => param.id).filter((parameter) => Object.keys(errors).includes(parameter)),
    }))
  )};`

Reply:
I tried but it doesn't spread the array correctly :slightly_frowning_face: